graylog2-web-interface/src/views/components/widgets/CopyToDashboardForm.test.jsx

```diff
@@ -74,16 +74,19 @@ describe('CopyToDashboardForm', () => {
     expect(onSubmit).toHaveBeenCalledWith('widget-id', 'view-1');
   });
 
-  it('should handle search of dashboards', () => {
+  it('should query for all dashboards & specific dashboards', () => {
     DashboardsActions.search = mockAction(jest.fn(() => Promise.resolve()));
-    const { getByPlaceholderText, getByText } = render(<CopyToDashboardForm dashboards={dashboardState}
-                                                                            widgetId="widget-id"
-                                                                            onCancel={() => {}}
-                                                                            onSubmit={() => {}} />);
+    const { getByPlaceholderText, getByText } = render(
```

Comment:
It started to fail once I made the changes. After pairing with @kmerz it seems that this test should probably not be passing in master because the `search` function should be called twice due to the way `useEffect` works.

So, now it properly checks the times its been run and passes the test.